ios/brave-ios/Sources/Onboarding/Welcome/WelcomeViewController.swift

```diff
@@ -405,26 +405,31 @@ public class WelcomeViewController: UIViewController {
     controller.calloutView.isLoading = true
     
     let attributionManager = controller.attributionManager
+    var featureLinkType: FeatureLinkageType?
```

Comment:
Move this into the `Task` so you are not breaking sendability (will be an error with strict swift concurrency on)

Reply:
Sounds good, good tip 
https://github.com/brave/brave-core/pull/22277/commits/23484a54c4ba3e1e15a34d22bbb654ecba009345